acos_client/v30/slb/virtual_port.py

```diff
@@ -152,7 +152,7 @@ def _set(
         url = self.url_server_tmpl.format(name=virtual_server_name)
 
         aflex_scripts = kwargs.get("aflex-scripts", None)
-        if aflex_scripts:
+        if aflex_scripts is not None:
```

Comment:
The difference is important.
IF returns False if length of list is 0. Which is not expected this case.
We explicitly expect if aflex_script is None.